src/visitpy/common/Logging.C

```diff
@@ -1424,7 +1430,7 @@ static std::string log_QueryRPC(ViewerRPC *rpc)
             if (numPrinted > 0)
                 s += ", ";
             s += "vars=";
-            s += queryParams.GetEntry("vars")->ConvertToString();
+            s += queryParams.GetEntry("vars")->ConvertToPythonTupleString();
```

Comment:
should this not be
```c++
if(queryParams.GetEntry("vars")->IsVector())
    s += queryParams.GetEntry("vars")->ConvertToPythonTupleString();
else
    s += queryParams.GetEntry("vars")->ConvertToString();
```
or are vars always a vector type?

Reply:
Interesting.  Just looked at logic elsewhere handling 'vars', and yes it can be a single string. Thanks! I will make the change.

Reply:
Actually, GUI is always setting vars into a stringVector, so the current Logging logic suffices.

Reply:
Will it always be that way? I feel it is more robust to always check in case things change in the future, but I defer to you.